Website/src/rendering/src/stories/Sections/Section.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import Section from '../../components/Page Content/Section';
+import SpeakersGrid, { SpeakersGridProps } from '../../components/Speakers/SpeakersGrid';
+import { GraphQLSpeaker } from 'src/types/speaker';
+
+export default {
+  title: 'Components/Sections/Section',
```

Comment:
There is already a Section Storybook file in `Website\src\rendering\src\stories\Page Content\Section.stories.tsx`. Please merge your variants in it and remove any duplicates.